test/factories/concept_exercises.rb

```diff
@@ -4,6 +4,7 @@
     uuid { SecureRandom.uuid }
     slug { 'datetime' }
     title { slug.titleize }
+    git_sha { "HEAD" }
```

Comment:
Once this gets close to finished, it might be worth updating these to be whatever the *actual* head commit in the fixture is.

Reply:
Indeed